resolwe_bio/tools/parse_cuffnorm.py

```diff
@@ -9,18 +9,22 @@
 
 parser = argparse.ArgumentParser(description="Parse Cuffnorm output")
 parser.add_argument('input_file', help="genes.fpkm_table file to parse")
-parser.add_argument('index', help="column index")
+parser.add_argument('sample_name', help="Sample name")
 parser.add_argument('-n', action='store_true', help="Normalized input")
 args = parser.parse_args()
 
 data = pd.read_csv(args.input_file, sep='\t')
-names = [i for i in list(data.columns.values)]
+names = [i.split('_Group')[0] for i in list(data.columns.values)]
+index = [i for i, name in enumerate(names) if args.sample_name == name]
+if len(index) > 1:
```

Comment:
@jvrakor Please move the check for duplicated sample names to the process code instead (maybe the first `for` loop on `cuffquant` objects). Use `re-error` there. In this way, you can detect the issue sooner and stop the process execution before any processing happens.